bittide/src/Bittide/ProcessingElement.hs

```diff
@@ -35,33 +38,34 @@ data PeConfig nBusses where
     InitialContent depthD (Bytes 4) ->
     PeConfig nBusses
 
--- | 'Contranomy' based RV32IMC core together with instruction memory, data memory and
+-- | VexRiscV based RV32IMC core together with instruction memory, data memory and
 -- 'singleMasterInterconnect'.
 processingElement ::
   forall dom nBusses .
   ( HiddenClockResetEnable dom
   , KnownNat nBusses, 2 <= nBusses, CLog 2 nBusses <= 30) =>
   PeConfig nBusses ->
   Circuit
-    ()
-    ( Vec (nBusses-2) (Wishbone dom 'Standard (MappedBusAddrWidth 32 nBusses) (Bytes 4))
-    )
-processingElement (PeConfig memMapConfig initI initD) = circuit $ do
-  (iBus0, dBus) <- rvCircuit (pure low) (pure low) (pure low)
+    (Jtag dom)
+    (Vec (nBusses-2) (Wishbone dom 'Standard (MappedBusAddrWidth 32 nBusses) (Bytes 4)))
+processingElement (PeConfig memMapConfig initI initD) = circuit $ \jtagIn -> do
+  (iBus0, dBus0) <- rvCircuit (pure low) (pure low) (pure low) -< jtagIn
+  iBus1 <- setName @"instr_bus" (ilaWb 2 D4096) -< iBus0
+  dBus1 <- setName @"data_bus" (ilaWb 2 D4096) -< dBus0
   ([iMemBus, dMemBus], extBusses) <-
-    (splitAtC d2 <| singleMasterInterconnect memMapConfig) -< dBus
+    (splitAtC d2 <| singleMasterInterconnect memMapConfig) -< dBus1
   wbStorage initD -< dMemBus
-  iBus1 <- removeMsb -< iBus0
-  wbStorageDPC initI -< (iBus1, iMemBus)
+  iBus2 <- removeMsb -< iBus1 -- XXX: <= This should be handled by an interconnect
```

Comment:
Let's make an issue to change the way we deal with memory busses and memories

Reply:
The interconnect usually deals with this, but because we have two busses and the instruction bus is (was?) not hooked up to an interconnect itself, this is an ad-hoc hack. I guess we could add an interconnect that maps to the same address and a bunch of dummy components?

Reply:
I think an `m` masters to `n` slaves interconnect would be a good solution, but I'd like an issue aimed at redesigning the layout of memories and busses.

Reply:
I'll leave this comment open and create an issue after merging